### PR TITLE
Issue #41. Create updated the readme.md file for the documentation

### PR DESCRIPTION
**Pull request by nhiieL**
of the drupalcampcebu2018 project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,114 @@
...
+```sh
+docker-compose run --rm php_nginx drush help
+docker-compose run --rm php_nginx drush updb
+docker-compose run --rm php_nginx drush cr
+docker-compose run --rm php_nginx drush uli
+```
+### CSS Editing
+Right now, all the CSS from `themes/custom/drupalcamp_cebu_2018/css` is directly ***Symlinked*** into `static/public/css/`. When you modify the CSS, do it in the ***Sass*** folder found in `static/resources/sass`.
+When first compiling ***Sass***, run `npm install` and `gulp` within the `static` folder. You may need to execute `gulp` from within `static` folder when you want to Sassify the CSS, or better yet, execute `gulp watch`.
+## Migration
```
Comment (Luukyb) [inline README.md]: This does not apply.

File: README.md [hunk 2]
```diff
@@ -1 +1,114 @@
...
+* [Docker](https://docs.docker.com/engine/installation/)
+* [Docker Compose](https://docs.docker.com/engine/installation/)
+## Local setup with Expresso PHP
+If it doesn't exists, create a folder docker in your home directory. This is where you will keep all your docker projects.
+Clone Expresso PHP in ~/docker
+```
+$ cd ~/docker
+$ git clone https://github.com/expresso-php/expresso-php.git drupalcampcebu2018
+```
+### Set Expresso PHP to match PROD: Nginx with PHP 7.0
```
Comment (Luukyb) [inline README.md]: Not applicable. We are using PHP 7.2.

Comment (nhiieL) [reply README.md]: kuya luc when I run php -v, it says that PHP 7.0.25. 
What do you mean we are using PHP 7.2 ?

Comment (nhiieL) [reply README.md]: phpversion

Comment (Luukyb) [reply README.md]: Hi @nhiieL , this is your laptop's PHP. If you /bin/bash into the containers and run the same command you should see PHP 7.2.

**Pull request merged**